test/unit/PanoViewer/PanoViewer.spec.js

```diff
@@ -245,6 +245,24 @@ describe("PanoViewer", function() {
 		});
 	});
 
+	describe("#setter/getter", function() {
```

Comment:
I did't understand this test describe block. because it doesn't have `it` block

Reply:
Oh, I'll remove that.